components/brave_rewards/resources/page/reducers/wallet_reducer.ts

```diff
@@ -201,6 +201,7 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
         ui.walletServerProblem = true
       } else if (status === 24) { // on ledger::Result::EXPIRED_TOKEN
         chrome.send('brave_rewards.getExternalWallet', ['uphold'])
+        state.balance.total = action.payload.balance.total || 0
```

Comment:
Will this end up displaying the user's token balance only (their "total" minus uphold)? If so, would that be confusing for users?

Reply:
it should not be as we just disconnected wallet as well, so they don't have any reference to uphold anymore until they connect again